content/posts/newsletter-november-2024.md

```diff
@@ -0,0 +1,35 @@
+title: Read the Docs Newsletter - November 2024
+date: November 10, 2024
+description: Company updates and new features from the last month, current focus, and upcoming features.
+category: Newsletter
+tags: newsletter, community
+authors: Eric Holscher
+status: published
+image: /images/november-2024.jpg
+image_credit: Photo by <a href="https://unsplash.com/@hermanndoyo?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Doyo Hermann</a> on <a href="https://unsplash.com/photos/a-scenic-view-of-a-town-surrounded-by-mountains-y1VQvwSqYOE?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Unsplash</a>
+
+
+## News and updates
```

Comment:
Now notification on latest and non-stable versions can be enabled/disabled independently.
https://github.com/readthedocs/readthedocs.org/pull/11744

Reply:
All the `build.tools` versions were updated.
https://github.com/readthedocs/readthedocs-docker-images/pull/209